v3/plugins/eclipse/gwt/next/meta.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 publisher: eclipse
 name: gwt
-version: 1.0.0
+version: next
```

Comment:
Why next? Maybe it should be renamed accordingly to docker image version `nightly`?